tests/rust_usage_test/tests/build_script_test.rs

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2018 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+extern crate syn;
+
+use std::{env, fs, path::Path};
+
+#[test]
+fn build_script_wrapper() {
```

Comment:
@rjsberry WDYT about comparing that the contents are the same as `monster_test_generated.rs`?

Reply:
Sounds reasonable. I've run a quick check locally and am seeing quite a few differences. When and how is `monster_test_generated.rs` generated?

Reply:
@rjsberry It's the result of `generate_code.sh`